src/polyswarm/client/polyswarm.py

```diff
@@ -128,11 +128,7 @@ def polyswarm(ctx, api_key, api_uri, output_file, output_format, color, verbose,
     if ctx.invoked_subcommand is None:
         return
 
-    # only allow color for stdout
-    if output_file is not None:
-        color = False
-    else:
-        output_file = click.get_text_stream('stdout')
+    output_file = output_file or click.get_text_stream('stdout')
```

Comment:
As the option is a click.File, setting the default to `'-'` would automatically select the stdout.
From: https://click.palletsprojects.com/en/7.x/arguments/#file-arguments
>  Command line tools are more fun if they work with files the Unix way, which is to accept - as a special file that refers to stdin/stdout.
> 
> Click supports this through the click.File type which intelligently handles files for you. It also deals with Unicode and bytes correctly for all versions of Python so your script stays very portable.
```